tests/test_template/test_policy_guidance.py

```
@@ -0,0 +1,153 @@
+        (
+            "must accept an injected environment: `mockable::Env`",
+            "`mockable::DefaultEnv` supplied at the production composition root",
+            "`mockable::MockEnv` in tests",
+            "`assert_cmd` may configure",
+            "`Command::env`/`Command::env_clear`",
+            "Subprocess isolation is the sole exemption",
+            "Process-wide locks are not an escape hatch",
+            "shared `Mutex`, `OnceLock`, or `serial_test` attribute",
+            "Clippy warnings MUST be disallowed",
+            'RUSTDOCFLAGS="$(RUSTDOC_FLAGS)" '
+            'RUSTFLAGS="$(DEV_RUST_FLAGS)" cargo doc --no-deps',
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)